### PR TITLE
allow running tests with intellij IDEA

### PR DESCRIPTION
**Pull request by akostadinov**
**What this PR does / why we need it**:

Allow running tests with Intellyj IDEA after git clean clone.

- custom reporters are not supported by IDEA
- without the /tmp/metrics directory many tests fail, so ensure it exists after clone

**Verification steps** 

1. git clone porta
2. import in IDEA
3. run some unit tests

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,7 +14,6 @@
 .deploy_info
 .ruby-gemset
 .ruby-version
-.tool-versions
```
[the remark this answers is not in the document]
Comment (akostadinov) [reply .gitignore:17]: It is a redundant entry

File: .gitignore [hunk 2]
```diff
@@ -98,6 +97,7 @@ C:\\nppdf32Log\\debuglog.txt
 /public/system
 /public/test*.html
 /tmp/*
+!/tmp/metrics
```
[the remark this answers is not in the document]
Comment (akostadinov) [reply .gitignore:100]: wait, this was not about tests or IDEA, I forgot and spent far too long to figure that out now :/

In fact there is an error in server logs (and IIRC sidekiq in certain conditions that I didn't reproduce).

```
D, [2021-07-19T15:49:38.614887 #575377] DEBUG -- : error No such file or directory @ rb_sysopen - /home/avalon/ws/repos/porta/tmp/metrics/histogram_575377-0.db backtrace /home/avalon/.asdf/installs/ruby/2.5.9/lib/ruby/gems/2.5.0/gems/prometheus-client-mmap-0.11.0/lib/prometheus/client/helper/file_locker.rb:13:in `initialize'
/home/avalon/.asdf/installs/ruby/2.5.9/lib/ruby/gems/2.5.0/gems/prometheus-client-mmap-0.11.0/lib/prometheus/client/helper/file_locker.rb:13:in `open'
/home/avalon/.asdf/installs/ruby/2.5.9/lib/ruby/gems/2.5.0/gems/prometheus-client-mmap-0.11.0/lib/prometheus/client/helper/file_locker.rb:13:in `block in lock_to_process'
/home/avalon/.asdf/installs/ruby/2.5.9/lib/ruby/gems/2.5.0/gems/prometheus-client-mmap-0.11.0/lib/prometheus/client/helper/file_locker.rb:9:in `synchronize'
/home/avalon/.asdf/installs/ruby/2.5.9/lib/ruby/gems/2.5.0/gems/prometheus-client-mmap-0.11.0/lib/prometheus/client/helper/file_locker.rb:9:in `lock_to_process'
/home/avalon/.asdf/installs/ruby/2.5.9/lib/ruby/gems/2.5.0/gems/prometheus-client-mmap-0.11.0/lib/prometheus/client/helper/mmaped_file.rb:45:in `block in ensure_exclusive_file'
...
```

[the remark this answers is not in the document]
Comment (hallelujah) [reply .gitignore:100]: Ah but how did you get this? I do not set `prometheus_multiproc_dir` environment variable and I have no issue

[the remark this answers is not in the document]
Comment (akostadinov) [reply .gitignore]: Thank you for pointing that out. I didn't notice setup scripts set this directory and it is not set by porta OOB. I removed this commit and will consider a proper way to fix setup scripts.

File: test/test_helper.rb [hunk 3]
```diff
@@ -33,7 +33,10 @@
 require 'minitest/reporters'
 
 junit = MiniTest::Reporters::JUnitReporter.new([junit_reporter_path, Process.pid].compact.join('-'))
-MiniTest::Reporters.use!([junit, MiniTest::Reporters::DefaultReporter.new])
+unless ENV['RM_INFO']
```
Comment (hallelujah) [inline test/test_helper.rb:36]:
```suggestion
# this environment variable is set by Intellij IDEA which is not compatible with those reporters
unless ENV['RM_INFO']
```

**Review by hallelujah — CHANGES_REQUESTED** [review]

**Review by hallelujah — APPROVED** [review]

**Pull request merged**